parsl/executors/high_throughput/process_worker_pool.py

```diff
@@ -576,7 +576,7 @@ def worker(
     # If desired, set process affinity
     if cpu_affinity != "none":
         # Count the number of cores per worker
-        avail_cores = sorted(os.sched_getaffinity(0))  # Get the available processors
+        avail_cores = sorted(os.sched_getaffinity(0))  # Get the available threads
```

Comment:
I feel like this vocabulary should be made more consistent: cores/threads/processors are used apparently interchangeably(?) here, where one word might do better: mostly this code uses `cores`

Reply:
This call gets the available hardware threads on the node, not the available physical cores, despite the variable name.  It's unfortunate that `cores` is used as it is in the code because for machines that support multiple hardware threads per core (e.g. many hpc systems) this makes the code confusing.  I don't work with this code often enough to remember what Parsl is doing here always and every time I look at it I have to remind myself what's going on.  Some kind of comment explaining this would be helpful here.